browser/src/Services/QuickOpen.ts

```diff
@@ -195,12 +164,26 @@ export class QuickOpen {
                 nodir: true,
                 ignore: Config.instance().getValue("oni.exclude"),
             }, (_err: any, files: string[]) => {
+                Log.error(_err)
+                if (!files) {
+                    this._loadDefaultMenuItems()
+                    this._showMenuFromQuickOpenItems(this._loadedItems)
+                }
+
                 files.forEach( (f: string) => {
```

Comment:
Is it possible that `files` can be null/undefined here? Or is it always an empty array? I'm wondering if this should be in an `else` block, otherwise seems like we could get an exception on the `forEach` call.

Reply:
That's a good catch @extr0py . As I was messing around with that block, I never saw the case of an error being thrown if the `!files` block was entered. Nonetheless, I think you're right. It would be better to be sure of that happening with an `else` check. I'll update the PR later today (LA time) and resubmit.